studio/components/layouts/ProjectLayout/LayoutHeader/FeedbackDropdown/FeedbackWidget.tsx

```diff
@@ -118,7 +118,7 @@ const FeedbackWidget = ({
           pathname: router.asPath,
         })
         setFeedback('')
-        ui.setNotification({ category: 'success', message: 'Feedback sent. Thank you!' })
+        ui.setNotification({ category: 'success', message: 'Feedback sent. Thank you!\n\nKindly be aware that we do not provide responses to feedback. If you require assistance or a reply, kindly consider submitting a support ticket.' })
```

Comment:
2x on "kindly" 
Maybe — "Please be aware that we ......, consider submitting a support ticket"

Reply:
It's the copy police, hide your kids, hide your wife